lib/net/peer.js

```diff
@@ -285,7 +296,19 @@ class Peer extends EventEmitter {
     this.address.services = 0;
     this.outbound = false;
     this.connected = false;
-    this.brontide.accept(socket, this.identityKey);
+
+    switch (this.type) {
+      case this.options.peerTypes.STANDARD:
```

Comment:
Is there a reason these constants reside on the options object? I feel like they should just be global constants.

Reply:
It was so that they could be shared between the `Pool` and `Peer` files easily but that could also be done by exposing them as a static `Peer.peerTypes`